src/service/apply/applyLoan.ts

```
@@ -0,0 +1,75 @@
+import { Response } from 'express';
+import { BasicResponse, AuthenticatedRequest } from '../../types';
+import { ApplyLoanRequest } from '../../types/apply';
+import { prisma } from '../../config/prisma';
+
+export const applyLoan = async (req: AuthenticatedRequest<{}, {}, ApplyLoanRequest>, res: Response<BasicResponse>) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)